pipelines.md

```diff
@@ -81,17 +83,73 @@ The default tasks and pipelines can be updated by forking the Kabanero Pipelines
 
 ### Tasks
 
-- [build-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/build-task.yaml)
+- [validate-active-stack-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/validate-active-stack-task.yaml)
+
+This task will validate the stack is allowed to build an deploy on the cluster.  It checks the digest of the stack image specified in the `.appsody-config.yaml` of the project and validates it matches the digest of the stack image that's active on the collection.  If they do not match, the pipeline will fail and none of the other steps will execute.
```

Comment:
Change
will validate
to
validates

change
build an deploy
to
build and deploy

Change
the `.appsody-config.yaml` of
to
the `.appsody-config.yaml` file of

Change
validates it matches
to
validate that it matches

"active on the collection."  <-- I thought we weren't using "collection" anymore?

Change 
If they do not match, the pipeline will fail and none of the other steps will execute.
to
If the images do not match, the pipeline fails and none of the other steps execute.